mixer/adapter/solarwinds/config/config.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package adapter.solarwinds.config;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "config";
+option (gogoproto.goproto_getters_all) = false;
+option (gogoproto.equal_all) = false;
+option (gogoproto.gostring_all) = false;
+
+message Params {
```

Comment:
Consider adding comments for each field and what is the default values for those fields. Right now it is unclear to me  how are these configs used and why is the instance object directly not sufficient to be consumed by the adapter.

Reply:
Will do.

Reply:
Sorry I forgot to mentioned earlier, it will be ideal to add an yaml example right on top of the param message (I don't think we do this for other adapters, but it is definitely a huge help for config authors)